R/reactives.R

```diff
@@ -1492,8 +1603,12 @@ reactiveTimer <- function(intervalMs=1000, session = getDefaultReactiveDomain())
 #' }
 #' @export
 invalidateLater <- function(millis, session = getDefaultReactiveDomain()) {
+
   force(session)
-  ctx <- .getReactiveEnvironment()$currentContext()
+
+  ctx <- getCurrentContext()
+  rLog$invalidateLater(ctx$id, millis, session)
```

Comment:
Won't you want access to `ctx$.reactId` as well, when you eventually get around to putting in some kind of visual effect?

Reply:
Fixed in fe679b5